docssrc/src/argument_entities.md

```diff
@@ -71,6 +71,49 @@ The `PlayerArgument` class is very similar _(almost identical)_ to `EntitySelect
 >
 > The `PlayerArgument` internally uses the `GameProfile` class from Mojang's authlib, which means that this argument has a slight performance overhead compared to using `EntitySelectorArgument.OnePlayer`
 
+<div class="example">
+
+### Example - Warp command
+
+When registering a `PlayerArgument` you might notice that it includes `Entity Selectors` (`@a`, `@e`, `@r`, etc.). If you want to avoid those, you can use safe argument suggestions to only suggest the player names. For this example, let us create a /warp command:
```

Comment:
```suggestion
When registering a `PlayerArgument` you might notice that it includes `Entity Selectors` (`@a`, `@e`, `@r`, etc.). If you want to avoid those, you can use argument suggestions to only suggest the player names. For this example, let us create a /warp command:
```

As this is not specific to safe suggestions, it might be nicer to phrase it this way.

Reply:
Changed it